hooks/tk-multi-workfiles2/scene_operation_tk-nuke.py

```diff
@@ -331,10 +332,21 @@ def _scene_operation_hiero_nukestudio(
         """
         import hiero
 
+        engine = sgtk.platform.current_engine()
+        engine_context = engine.context
+        context_str = engine_context.serialize(engine_context)
+
         if operation == "current_path":
             # return the current script path
             project = self._get_current_hiero_project()
             curr_path = project.path().replace("/", os.path.sep)
+            if curr_path:
+                # switch file-save window context to selected project context
+                project = self._get_current_hiero_project()
+                tag_object = self.get_tag_object(project)
+                context_str = tag_object.note()
```

Comment:
This should be `selected_context_str` to be within this scope.

Reply:
Addressed comment.